### PR TITLE
config.tf: update tco version and correct tectonic version

### PR DESCRIPTION
**Pull request by rithujohn191**

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -22,7 +22,7 @@ variable "tectonic_container_images" {
     console                      = "quay.io/coreos/tectonic-console:v1.5.1"
     identity                     = "quay.io/coreos/dex:v2.4.1"
     kube_version_operator        = "quay.io/coreos/kube-version-operator:7da46d189c36092f43d07ca381a61897402fa13c"
-    tectonic_channel_operator    = "quay.io/coreos/tectonic-channel-operator:15c001bd7c008a04394390d08ac71046e723ac48"
+    tectonic_channel_operator    = "quay.io/coreos/tectonic-channel-operator:v0.2.6"
```
Comment (yifan-gu) [inline config.tf]: It's a semver, so should just be `0.2.6` without `v`

File: modules/update-payload/payload.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
                     }
                   }
                 ],
-                "image": "quay.io/coreos/tectonic-channel-operator:15c001bd7c008a04394390d08ac71046e723ac48",
+                "image": "quay.io/coreos/tectonic-channel-operator:v0.2.6",
```
Comment (Quentin-M) [inline modules/update-payload/payload.json]: same here

**Comment by Quentin-M** [comment]
note that you will need KVO too.

File: config.tf (hunk 1 above)
Comment (rithujohn191) [inline config.tf]: done

File: modules/update-payload/payload.json (hunk 2 above)
Comment (rithujohn191) [inline modules/update-payload/payload.json]: done

**Comment by rithujohn191** [comment]
waiting for info on if we need a new kvo version

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
The upgrade test is failing because I forgot to change the domain name in test. Fixed, and retriggering.

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by rithujohn191** [comment]
Updated TCO to 0.3.0

**Comment by rithujohn191** [comment]
Waiting for KVO bump

File: modules/update-payload/payload.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
                     }
                   }
                 ],
-                "image": "quay.io/coreos/tectonic-channel-operator:15c001bd7c008a04394390d08ac71046e723ac48",
+                "image": "quay.io/coreos/tectonic-channel-operator:0.3.0",
```
Comment (yifan-gu) [inline modules/update-payload/payload.json]: 0.3.1 is out now

**Comment by derekparker** [comment]
KVO newest version tag is `v1.6.2`.

**Comment by rithujohn191** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from master

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Comment by yifan-gu** [comment]
test upgrade from v1.6.2-tectonic.1-rc.4

**Pull request merged**